localisation/english/russia2_localisation_l_english.yml

```
@@ -7318,6 +7345,54 @@
+ russia2_russia.676.t:0 "Neverending Revolution Successful"
+ russia2_russia.676.d:0 "Our recent wars have been a success! Thus some of the people who have helped achieve victory have requested to join us."
+ russia2_russia.676.a:0 "Of Course."
+ russia2_russia.676.b:0 "They are better of on their own."
+ will_of_the_tsar:0 "Will of the Tsar"
+ will_of_the_tsar_desc:0 "The will of the Tsar is guaranteed as long as their power is unending."
+ imperial_council:0 "Imperial Council"
+ russia2_russia.677.t:0 "The Great Imperial War"
+ russia2_russia.677.d:0 "The Great Imperial War is getting ready to begin, the reign of monarchs is about return! However despite the excitement in the air, it is up to the Tsar to truly decide whether or not Russia is ready for such a massive conquest."
+ russia2_russia.677.a:0 "Russia is ready."
+ russia2_russia.677.b:0 "Fighting now would mean certain death."
+ russia2_russia.678.d:0 "The monarchies great strike back has begun and so has Russia's conquest of the world!" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)